migrations/1605181184453-users.js

```diff
@@ -0,0 +1,93 @@
+var db = require('./db')
```

Comment:
uppercase on this file as well... a simple find and replace on vscode should do the job